packages/core/src/rum/instrumentation/resourceTracking/implementation/DatadogRumResource/__tests__/internalDevResourceBlocklist.test.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+import { filterDevResource } from '../internalDevResourceBlocklist';
+
+import { ResourceMockFactory } from './__utils__/ResourceMockFactory';
+
+const resourceMockFactory = new ResourceMockFactory();
+
+describe('internalDevResourceBlocklist', () => {
+    describe('filterDevResource', () => {
```

Comment:
I suggest also adding the test(s) checking that `resource` is returned when URL is not in the dev list